aws-janitor/resources/list.go

```diff
@@ -80,6 +80,7 @@ var RegionalTypeList = []Type{
 	LaunchConfigurations{},
 	LaunchTemplates{},
 	Instances{},
+	VPCEndpoints{},
```

Comment:
Fix the order issue here, the VPC endpoints should be cleaned ahead of network interfaces and security groups, other wise, some dependency errors might happen.